dotCMS/src/main/java/com/dotcms/rest/api/v1/experiment/ExperimentResource.java

```diff
@@ -0,0 +1,358 @@
+            + "                window.dispatchEvent(event);\n"
+            + "              } \n"
+            + "           });\n"
+            + "}\n";
+
+    private final String scriptTraffic = "console.log('traffic_scripts');\n"
+            + "\n"
+            + "window.addEventListener(\"init_custom\", function (event) {\n"
+            + "    console.log('traffic init_custom', event.detail.experiment.variant);\n"
+            + "    \n"
+            + "    if (!window.location.href.includes('redirect=true')) {\n"
+            + "        window.location = event.detail.experiment.variant.url;\n"
+            + "    }\n"
+            + "});";
+
+    private final String scriptMetricts = "var initEvent = new Promise(function(resolve) {\n"
+            + "    window.addEventListener(\"init_custom\",resolve,false);\n"
+            + "});\n"
+            + "var loadEvent = new Promise(function(resolve) {\n"
+            + "    window.addEventListener(\"load\", resolve, false);\n"
+            + "});\n"
+            + "\n"
+            + "Promise.all([initEvent, loadEvent]).then(function(data) {\n"
+            + "%s"
+            + "});";
+
+    @GET()
+    @Path("/js/experiments.js")
+    @Produces({"application/javascript"})
+    public String loadJS(@Context final HttpServletRequest request,
+            @Context final HttpServletResponse response)
+            throws DotSecurityException, DotDataException {
+
+        final Optional<Cookie> experimentCookie = getExperimentCookie(request);
+
+        if (experimentCookie.isPresent()) {
+            final Optional<Experiment> experiment = ExperimentFactory.getExperiment(experimentCookie.get().getValue());
+
+            if (!experiment.isPresent()) {
+                throw new NotFoundException();
+            }
+
+            final Collection<AnalyticEvent> events = experiment.get().getEvents();
+            final StringBuffer metricsJsCode = new StringBuffer();
+
+            for (final AnalyticEvent event : events) {
+                final AnalyticEventTypeHandler analyticEventTypeHandler = AnalyticEventTypeHandlerManager.INSTANCE.get(
+                        event.getEventKey());
+
+                final String jsCode = analyticEventTypeHandler.getJSCode(event.getParameters());
+                metricsJsCode.append(jsCode);
+            }
+            return scriptTraffic + String.format(scriptMetricts, metricsJsCode) + scriptInit;
+        } else {
+            throw new IllegalStateException();
+        }
+
+    }
+
+    private Optional<Cookie> getExperimentCookie(HttpServletRequest request) {
+        final Cookie[] cookies = request.getCookies();
+
+        for (Cookie cookie : cookies) {
+            if (cookie.getName().equals(EXPERIMENT_COOKIE_NAME)) {
+                return Optional.of(cookie);
+            }
+        }
+
+        return Optional.empty();
+    }
+
+    @GET
+    @Path("/result/{key}")
```

Comment:
Change path to `{key}/results` to the final path looks like `experiments/{key}/results`